### PR TITLE
Update Making release section. Closes #167

### LINKED ISSUE #167: Improve release page
https://accumulo.apache.org/contributor/making-release

There is a lot of good info on this page but it is a little hard to follow.  A nice TODO list at the top of the page, linking to each section would be great.  @ctubbsii [[VOTE] Apache Accumulo 1.9.3-rc3](http://apache-accumulo.1065345.n5.nabble.com/VOTE-Apache-Accumulo-1-9-3-rc3-tp23293p23317.html) email on the release vote made me think of this.

### PR DESCRIPTION
**Pull request by milleruntime**

### CODE REVIEW

File: contributor/making-release.md [hunk 1]
```diff
@@ -6,7 +6,43 @@ redirect_from:
   - /governance/releasing
 ---
 
-This is a guide for the creation of a release of Apache Accumulo. 
+Follow these steps to make a release of Apache Accumulo. 
+
+1. [Setup]
```
Comment (keith-turner) [inline contributor/making-release.md]: I wonder if this would work `[Setup](#setup)` and then would not need to define links below.

**Comment by milleruntime** [comment]
@ctubbsii I copied over some of your bullet points from your email and the ones I weren't sure where to put are in the section labeled TODO.

File: contributor/making-release.md [hunk 2]
```diff
@@ -6,7 +6,43 @@ redirect_from:
   - /governance/releasing
 ---
 
-This is a guide for the creation of a release of Apache Accumulo. 
+Follow these steps to make a release of Apache Accumulo. 
+
+1. [Setup](#setup)
+2. [Update the CHANGES file](#update-the-changes-file)
```
Comment (ctubbsii) [inline contributor/making-release.md]: We don't have a CHANGES file any more.

File: contributor/making-release.md [hunk 3]
```diff
@@ -6,7 +6,43 @@ redirect_from:
...
+3. [Triage Issues](#triage-issues)
+4. [Create the candidate](#create-the-candidate)
+5. [Vote](#vote)
+6. [Promote the artifacts](#promote-the-artifacts)
```
Comment (ctubbsii) [inline contributor/making-release.md]: I would probably create a "post-release tasks" section starting at after the vote.

File: contributor/making-release.md [hunk 4]
```diff
@@ -16,12 +52,15 @@ There are number of things that are required before attempting to build a releas
...
    the maven-release-plugin and staging the release candidate).
-4. Update the CHANGES file so that it's in sync with Jira (manual process).
-5. Ensure that you have a texlive distribution installed so you are able to build the documentation.
-6. Have a clean workspace before starting.
+4. Ensure that you have a texlive distribution installed so you are able to build the documentation.
+5. Have a clean workspace before starting.
 
 Given all of this, it's recommended that you only attempt making a release from a GNU/Linux machine.
 
+## Update the CHANGES file
```
Comment (ctubbsii) [inline contributor/making-release.md]: This file doesn't exist.

File: contributor/making-release.md [hunk 5]
```diff
@@ -16,12 +52,15 @@ There are number of things that are required before attempting to build a releas
 2. Ensure that you're using the correct major release of Java (check javadoc too).
 3. Ensure that you're building Apache Accumulo with a username that has the same name as your Apache ID (this is due to
    the maven-release-plugin and staging the release candidate).
-4. Update the CHANGES file so that it's in sync with Jira (manual process).
-5. Ensure that you have a texlive distribution installed so you are able to build the documentation.
-6. Have a clean workspace before starting.
+4. Ensure that you have a texlive distribution installed so you are able to build the documentation.
```
Comment (ctubbsii) [inline contributor/making-release.md]: We don't use LaTeX any more. We use asciidoc for 1.9, and 2.0 has docs directly on the website.

File: contributor/making-release.md [hunk 6]
```diff
@@ -16,12 +52,15 @@ There are number of things that are required before attempting to build a releas
...
 
 Given all of this, it's recommended that you only attempt making a release from a GNU/Linux machine.
 
+## Update the CHANGES file 
+
+Update the CHANGES file so that it's in sync with Jira (manual process).
+
 ## Triage issues.
 
 Before creating a release candidate, all open issues with a fix version of the
```
Comment (ctubbsii) [inline contributor/making-release.md]: I don't think we need wording about JIRA or GitHub transition. 2.0 is the last release with issues tagged in JIRA still... we can just drop this wording, since it adds unnecessary verbosity.

File: contributor/making-release.md [hunk 7]
```diff
@@ -125,8 +167,12 @@ After a successful vote, [this website][website-repo] needs to be updated with t
   * Create a post in `_posts/release/` containing release notes
   * Remove previous bug-fix release (if applicable)
   * Update doap/accumulo.rdf
+  * complete release notes 
+  * update previous release notes (as archived or archived-critical) 
+  * update _config.yml 
+  * update _includes/nav.html
```
Comment (ctubbsii) [inline contributor/making-release.md]: Escape the underscores, because it's a markdown char. Could use backtick to quote the filenames.

File: contributor/making-release.md [hunk 8]
```diff
@@ -158,7 +204,7 @@ Once a collection is created for a major release, developers can make documentat
 
 **For 1.x minor & bugfix releases,** copy `accumulo_user_manual.html` generated for release to the `1.x/` directory in the [accumulo-website repo][website-repo].
 
-### Javadocs
+## Update Javadocs
 
 Javadocs are easy to update. Using the latest JDK7 or later (at least JDK 7u21
```
Comment (ctubbsii) [inline contributor/making-release.md]: We already require Java 8 to run the build, even for 1.9 branch, so this whole JDK7 stuff is irrelevant.

**Comment by milleruntime** [comment]
Thanks for the feedback.  I am still not sure where to incorporate the bullet points I added in the TODO section from the email.  I also think we could make the "Post Release Tasks" a lot more concise.
